k8s/cloud/base/ory_auth/kratos/kratos_deployment.yaml

```diff
@@ -137,11 +137,11 @@ spec:
         # yamllint disable rule:indentation
         command: ['sh', '-c', 'set -x;
           URL="${ADMIN_URL}/admin/health/ready";
-          until [ $(curl -m 0.5 -s -o /dev/null -w "%{http_code}" --cacert /certs/ca.crt ${URL}) -eq 200 ]; do
+          until [ $(curl -k -m 0.5 -s -o /dev/null -w "%{http_code}" --cacert /certs/ca.crt ${URL}) -eq 200 ]; do
```

Comment:
Hi @MrSupiri ! Since we're going with `-k`, let's remove the `--cacert` flag from these commands, since those will be unused.

Reply:
Fixed in https://github.com/pixie-io/pixie/pull/686/commits/05c28442037507652532f9c2fcdb6cda3e26e7b9